core/vm/evm.go

```diff
@@ -213,14 +213,19 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 		return nil, gas, 0, ErrInsufficientBalance
 	}
 	if addr.Equal(LockupContractAddresses[[2]byte(evm.chainConfig.Location)]) {
-		ret, err := RunLockupContract(evm, caller.Address(), &gas, input)
+		ret, err = RunLockupContract(evm, caller.Address(), &gas, input)
```

Comment:
This change is super subtle. Is this our intention?

Reply:
Yes

Reply:
This specific change likely won't make a difference, but the point of the PR is to ensure we don't overwrite named returns